### PR TITLE
Final tweaks

### PR DESCRIPTION
**Pull request by trangdata**
Ready for review.

### CODE REVIEW

File: content/20.results.md [hunk 1]
```diff
@@ -18,41 +17,18 @@ For each article, we extracted its last author's fore name and last name for ana
 
 Defining an appropriate background distribution is key to determining the extent to which representation diverges from that background.
 We were faced with multiple decisions as we sought to define a field-specific distribution.
```
Comment (cgreene) [inline content/20.results.md]: I think we should talk about these key decisions:
* Honorees vs honors, authors vs authorships? We chose honors/authorships and why.
* What is the background? All of science, journals, MESH? We chose MESH and why.
* Which authors are the senior ones eligible for authorship? Corresponding, last? We chose Last and why.
* How should we evaluate articles? As equal contributions? Weighted by citation? We chose equal and why.

In this section we would reason through each of the decisions without giving information about examining multiple options. Then as the final part, we could say that in multiple versions we evaluated different decisions (refer to methods) and results were consistent (refer to supplement).

Comment (cgreene) [reply content/20.results.md]: Then perhaps the title could be: "Defining an evaluation set and field-specific background"

Comment (trangdata) [reply content/20.results.md]: I made three of the four points in 551e5a7 and 876cb72. 
- Discussing equal weight on articles is difficult without examining the other option of weighting by citation. I find it really hard to write, especially in the new regression framework we have.

File: content/40.resource-availability.md [hunk 2]
```diff
@@ -1,8 +1,24 @@
-## Data and Resource Availability
+## Resource Availability
 
-This manuscript was written [openly on GitHub](https://github.com/greenelab/iscb-diversity-manuscript) using Manubot [@doi:10.1371/journal.pcbi.1007128].
-The Manubot HTML version is available under a Creative Commons Attribution (CC BY 4.0) License at <https://greenelab.github.io/iscb-diversity-manuscript/>.
-Our analysis of authors and ISCB-associated honorees is available under CC BY 4.0 at <https://github.com/greenelab/iscb-diversity>, with source code also distributed under a BSD 3-Clause License.
-Rendered Python and R notebooks from this repository are browsable at <https://greenelab.github.io/iscb-diversity/>.
-Our analysis of PubMed, PubMed Central, and author names relies on the Python pubmedpy package, developed as part of this project and available under a Blue Oak Model License 1.0 at <https://github.com/dhimmel/pubmedpy> and on [PyPI](https://pypi.org/project/pubmedpy/).
+### Lead Contact
+Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Casey Greene (greenescientist@gmail.com).
```
Comment (cgreene) [inline content/40.resource-availability.md]:
```suggestion
Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Casey Greene (casey.s.greene@cuanschutz.edu).
```
To comply with public records / FOIA requirements as a state employee I now use Anschutz for work-related emails 😁

File: content/20.results.md [hunk 3]
```diff
@@ -5,54 +5,49 @@ The ISCB Fellows set contained the complete set of Fellows named (2009--2019).
...
+](https://raw.githubusercontent.com/greenelab/iscb-diversity/d84aa95ea02fff615c89668a20a957298df855ed/figs/workflow.png){#fig:workflow width="90%"}
 
 ### Similar gender proportion between ISCB's honorees and the field
 
 We observed a gradual increase of the proportion of predicted female authorships, arriving at an average of approximately 28% in 2017-2019 (Fig. {@fig:gender_breakdown}, left).
 In recent years, ISCB Fellows and keynote speakers appear to have similar gender proportions compared to the population of authors published in computational biology and bioinformatics journals (averaged around 30% in the last three years, Fig. {@fig:gender_breakdown}, right).
-Examining each honor category, we observed an increasing trend of honorees who were women, especially in the group of ISCB Fellows (see [notebook](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1)), which markedly increased after 2015.
+Examining each honor category, we [observed](https://greenelab.github.io/iscb-diversity/10.visualize-gender.html#sup_fig_s1) an increasing trend of honorees who were women, especially in the group of ISCB Fellows, which markedly increased after 2015.
```
Comment (cgreene) [inline content/20.results.md:50]: Does this need to refer to the notebook number like the others do?

**Review by cgreene — APPROVED** [review]
One missed notebook name and updated email address

File: content/20.results.md (hunk 3 above)
Comment (trangdata) [reply content/20.results.md:50]: I don't think so. This is also very minor. My goal was for it to still make sense even when the hyperlink disappears.

Comment (cgreene) [reply content/20.results.md:50]: What about: "... we observed in [10.visualize-gender] an increasing..."? It feels like it should refer somewhere in the sentence if the hyperlink goes away.

Comment (trangdata) [reply content/20.results.md:50]: Sorry, I just now see this. OK I'll add in a separate commit.

Comment (trangdata) [reply content/20.results.md:50]: dde6d54a542ee8b05e58ca9cfc82e7340da98ae9

**Pull request merged**